### PR TITLE
feat: #989 add accessibility in buttons on Home page [Pavel Kladkevich]

### PR DESCRIPTION
**Pull request by kladweb**
Add aria-label to theme switch button and sidebar button.
The buttons that have text value do not require aria-label.

### CODE REVIEW

File: mw-webapp/src/component/themeSwitcher/ThemeSwitcher.tsx [hunk 1]
```diff
@@ -78,6 +78,7 @@ export const ThemeSwitcher = (props: ThemeSwitcherProps) => {
         className={clsx(styles.iconWrapper, props.className)}
         onClick={onChangeTheme}
         data-cy={props.dataCy}
+        aria-label={props.dataCy}
```
Comment (tritonJS826) [inline mw-webapp/src/component/themeSwitcher/ThemeSwitcher.tsx:81]: We will probably remove data-attriutes for testing purposes from production some day

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ export const SidebarTrigger = forwardRef((props: PropsWithChildren<SidebarTrigge
       <div
         ref={ref}
         role="button"
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx:28]: 1. Task is not about adding aria-label to the Sidebar Trigger
2. Please read more about aria-label. Need research which tet should be in the aria and which tags couldn't include aria

For example, https://doka.guide/a11y/aria-label/


Please write  other oficial documentation about aria

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx (hunk 2 above)
Comment (kladweb) [reply mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx:28]: I think I should have added the aria tag only to the theme toggle button. The text value should match the language of the application.
Also I need to research and write for which elements we should add arria label or text values for accessibility.
Am I thinking in the right direction ?

Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx:28]: > I think I should have added the aria tag only to the theme toggle button.
Yes

Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx:28]: > The text value should match the language of the application.

Need research. It's a task to read  in the docs which value  there should be

Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx:28]: > Also I need to research and write for which elements we should add arria label or text values for accessibility.

Research need all time to check issues (maybe it's not a bug or bug is deprecated and etc.

Comment (kladweb) [reply mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx]: My research: 
https://docs.google.com/document/d/1pBUqgu7fRuseeu_52bSyRhO3oeVFuPvZ7gwjV_h8Wvw/edit?usp=sharing

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/themeSwitcher/ThemeSwitcher.tsx (hunk 1 above)
Comment (kladweb) [reply mw-webapp/src/component/themeSwitcher/ThemeSwitcher.tsx]: Answered in dialogue with Ekaterina

**Pull request merged**